src/powershell/devcontainer-feature.json

```diff
@@ -10,7 +10,9 @@
             "proposals": [
                 "latest",
                 "none",
-                "7.1"
+                "7.4",
```

Comment:
Can we add an explicit test to validate these changes work for the issue mentioned in https://github.com/devcontainers/features/issues/1041? Thank you!

Reply:
I am going to add a test for that particular version.

Reply:
Added a new test with 2ea64a6b0df8859ae9c697885743a465970ad330